packages/web-app-files/src/helpers/resource/actions/transfer.ts

```diff
@@ -0,0 +1,182 @@
+import { Resource } from 'web-client'
+import { join } from 'path'
+import { SpaceResource } from 'web-client/src/helpers'
+import { ClientService } from 'web-pkg/src/services'
+import {
```

Comment:
Please clean up the imports. The three import statements in lines 5-10 can be merged into one import statement.